### DISCUSSION TITLE
Node requirements on network seems incorrect

### DISCUSSION THREAD

**Discussion by gftea** [aptos-labs/aptos-developer-discussions#500] [Dev Doc and Tutorial Feedback]
In node requirement, it says only 1Gbps network bandwidth is needed, I wonder if it should be 10Gbps instead of 1Gbps? https://aptos.dev/en/network/nodes/validator-node/node-requirements

**Comment by 0xbrw** [comment]
The official documentation specifies a networking bandwidth of 1Gbps should be sufficient. What kind of issue did you run into when running a node on 1Gbps?
